test/container/Dockerfile-Void

```
@@ -0,0 +1,43 @@
+FROM ghcr.io/void-linux/void-glibc-full
+
+RUN xbps-install -Syu \
+    asciidoc \
+    base-devel \
+    bash \
+    binutils \
+    btrfs-progs \
+    busybox \
+    cargo \
+    cpio \
+    cryptsetup \
+    curl \
+    dhclient \
+    dmraid \
+    dosfstools \
+    e2fsprogs \
+    edk2-ovmf \
+    eudev \
+    git \
+    gnupg \
+    gummiboot \
+    iputils \
+    kbd \
+    libkmod-devel \
+    linux \
+    lvm2 \
+    make \
+    mdadm \
+    mtools \
+    nbd \
+    NetworkManager \
+    ntfs-3g \
+    open-iscsi \
+    openssh \
+    parted \
+    pigz \
+    procps-ng \
+    qemu \
+    squashfs-tools \
+    sudo \
+    util-linux \
+    xz \
```

```suggestion
    xz \
    && rm -rf /var/cache/xbps
```
remove any cached package files to save a little space